src/nanotron/config/models_config.py

```diff
@@ -7,13 +7,60 @@
 class RandomInit:
     std: float
 
+@dataclass
+class MambaInit:
```

Comment:
If we place Mamba in /examples, then its logic should not reside in src/nanotron